getting-started/pull-request-lifecycle.rst

```diff
@@ -183,59 +183,93 @@ message. It is usually okay to leave that as-is and close the editor.
 See `the merge command's documentation <https://git-scm.com/docs/git-merge>`_
 for a detailed technical explanation.
 
-
 .. _good-prs:
 
-Making good PRs
+Making Good PRs
 ===============
 
-When creating a pull request for submission, there are several things that you
-should do to help ensure that your pull request is accepted.
-
-#. **Make your change against the right version of Python.** In general all
-   changes are made against the ``main`` branch first. This includes bug fixes.
-   After the change is merged there, it will be :ref:`ported back <branch-merge>`
-   to older :ref:`maintenance releases <branchstatus>` as well. That way we
-   ensure all affected versions are handled. Therefore, basing a new change
-   directly on a maintenance branch is only used in specific circumstances,
-   for instance when that change does not apply to ``main`` or the change
-   requires a different approach in an older Python version compared to
-   ``main``.
-
-#. **Make sure to follow Python's style guidelines.** For Python code you
-   should follow :PEP:`8`, and for C code you should follow :PEP:`7`. If you have
-   one or two discrepancies those can be fixed by the core developer who merges
-   your pull request. But if you have systematic deviations from the style guides
-   your pull request will be put on hold until you fix the formatting issues.
+When creating a pull request, following best practices ensures your contribution is **clear, maintainable, and easy to review**. A well-structured PR improves collaboration and speeds up the review process.
+
+1. **Use a Clear and Structured PR Title**
```

Comment:
This is general guidance, not specific to Python. I'm not sure it makes sense to give this sort of general guideline: the devguide is more helpful to developers who want to start contributing to Python if it talks specifically about what's special about this project.